src/Controls.js

```diff
@@ -0,0 +1,43 @@
+import React, { Fragment } from 'react'
+import styled, { withTheme } from 'styled-components'
+import { previous, next } from './updaters'
+
+const Button = styled.div([], {
+  cursor: 'pointer',
+  width: '64px',
+  height: '100vh',
+})
+const Previous = styled(Button)([], {
+  position: 'fixed',
+  top: 0,
+  left: 0,
+  bottom: 0,
+})
+const Next = styled(Button)([], {
+  position: 'fixed',
+  top: 0,
+  right: 0,
+  bottom: 0,
+})
+
+const Controls = ({ update, theme: { controls } }) =>
+  controls !== false ? (
```

Comment:
Wouldn't it be better to have this condition in the parent?

Reply:
That was my initial idea too but I wasn't sure about having the theme in the provider. Think it's better ?